Lib/test/test_clinic.py

```diff
@@ -1,16 +1,28 @@
-# Argument Clinic
-# Copyright 2012-2013 by Larry Hastings.
-# Licensed to the PSF under a contributor agreement.
-#
+"""
```

Comment:
Nit: IMO converting this to `__doc__` doesn't make it any better. I prefer to keep license headers as comments.

Reply:
ok, i reverted this change.